stardog-union/stardog-archetypes
# Commit for #1 - X3D Archetype

# Description

Added the X3D archetype schema with fixed on inappropriate domain literal values in owl:unionOf, and other syntax errors
Added a namespace file that included x3do, x3d, and dublin core
Added a README showcasing a schema level query and an instance query using one of the CAD examples

## Manual test plan:

Test 1:
Action: Create a database with the x3do archetype
Verify: database created with expected number of triples
Results:
```
──> stardog-admin db create -o database.archetypes="x3do" -n testx3db
Bulk loading data to new database testx3db.
Loaded 8,948 triples to testx3db from 2 file(s) in 00:00:01.869 @ 4.8K triples/sec.
Successfully created database 'testx3db'
```

Test 2:
Action: Create a database with teh x3do archetype and one other archetype
Verify: Database created with expected number of triples
Results:
```
└──> stardog-admin db create -o database.archetypes="x3do,cim" -n testcimx3db
Bulk loading data to new database testcimx3db.
Loaded 21,118 triples to testcimx3db from 4 file(s) in 00:00:01.745 @ 12.1K triples/sec.
Successfully created database 'testcimx3db'.
```

Test 3:
Action: Create a database with the x3do archetype
Verify: namespaces created
Results:
```
└──> stardog namespace list testx3db
+---------+-----------------------------------------------------+
| Prefix  |                      Namespace                      |
+---------+-----------------------------------------------------+
|         | http://api.stardog.com/                             |
| dc      | http://purl.org/dc/terms/                           |
| owl     | http://www.w3.org/2002/07/owl#                      |
| rdf     | http://www.w3.org/1999/02/22-rdf-syntax-ns#         |
| rdfs    | http://www.w3.org/2000/01/rdf-schema#               |
| stardog | tag:stardog:api:                                    |
| x3d     | http://www.web3d.org/specifications/x3d-4.0.xsd#    |
| x3do    | http://www.web3d.org/specifications/X3dOntology4.0# |
| xsd     | http://www.w3.org/2001/XMLSchema#                   |
+---------+-----------------------------------------------------+
```

Test 4:
Action: Create a database with the x3do archetype
Verify: Run the first example schema query in the README with reasoning
Results: Results as expected
Verify: Run the first example schema query in the README without reasoning
Results: Results as expected

Test 5:
Action: Create a database with the x3do archetype
Verify: Run the second example data query in the README with reasoning
Results: Results as expected
Verify: Run the second example data query in the README without reasoning
Results: Results as expected
